src/components/BorrowerProfile/LendCta.vue

```diff
@@ -798,6 +815,29 @@ export default {
 	},
 	mounted() {
 		this.createWrapperObserver();
+		const erlCampaign = this.$route.query.utm_campaign;
```

Comment:
Most of this logic can be moved to the utils file and shared in both components. The new method can just return the value needed for `selectedOption`